docs/examples/image_morphology/image_morphology.jl

```diff
@@ -0,0 +1,132 @@
+# ---
+# cover:  assets/morphology.gif
+# title: Morphological Operations
+# author: Ashwani Rathee
+# date: 2021-7-2
+# ---
+
+# Morphological image processing is a area of computer vision which pursue tophat
+# remove imperfections from binary and grayscale images. ImageMorphology.jl
+# holds a collection of non-linear operations related to the shape or
+# morphology of features in an image.
+
+# In this demonstration, we cover the 6 most common morphological operations:
+# - Erosion
+# - Dilation
+# - Opening
+# - Closing
+# - Tophat
+# - Bothat 
+# - Morphological Gradient
+# - Morphological Laplace
+
+# Let's import an image 
+
+using Images, ImageView, ImageMorphology, TestImages, MosaicViews
```

Comment:
- `MosaicViews` are provided by `Images` (and `ImageCore`).
- `ImageView` is not used

```suggestion
using ImageMorphology
using Images, TestImages
```